src/lib.rs

```diff
@@ -218,7 +218,7 @@ impl Colorable for DirEntry {
     }
 }
 
-const LS_COLORS_DEFAULT: &str = "rs=0:lc=\033[:rc=m:cl=\033[K:ex=01;32:sg=30;43:su=37;41:di=01;34:st=37;44:ow=34;42:tw=30;42:ln=01;36:bd=01;33:cd=01;33:do=01;35:pi=33:so=01;35:";
+const LS_COLORS_DEFAULT: &str = "rs=0:lc=\\033[:rc=m:cl=\\033[K:ex=01;32:sg=30;43:su=37;41:di=01;34:st=37;44:ow=34;42:tw=30;42:ln=01;36:bd=01;33:cd=01;33:do=01;35:pi=33:so=01;35:";
```

Comment:
I think this should be `\x1b` rather than `\\033`

Reply:
Oh yes, thanks. https://github.com/sharkdp/lscolors/commit/b643e5dcb44f0767d4978d9707bfa39a54b630d7